### ISSUE TITLE
Improve usage to note --domain is only for cf

### ISSUE THREAD

**Issue by chinangela** [cloudfoundry/bosh-bootloader#142]
When trying to create a new concourse, we tried to run `bbl create-lbs --type cf --domain some-domain`, expecting the command to create a hosted zone. However, the `--domain` flag is not supported for `create-lbs` with `type=concourse`. It would be great to have this explicitly noted somewhere, so a user does not think this is a bug.

Cheers,
Angela and @joachimvaldez

**Comment by genevievelesperance** [comment]
Clarified in bbl help.

create-lbs:
```
[--domain]          Creates a nameserver with a zone for given domain (supported when type="cf")
```

update-lbs:
```
[--domain]           Updates domain in the nameserver zone (supported when type="cf", optional)
```

**Comment by sjolicoeur** [comment]
@genevievelesperance  I've also encountered this, would it not be possible for the `--domain` switch to work for both type of load balancers? 

I've recently tried to convert a CF install to a Concourse install, the instructions indicated just a few steps:
- `bosh deploy`
- destruction of the cf load balancer
- creation of a Concourse load balancer.

afterward we were left with quite a few manual steps on GCP, with no success. Compared to the near flawless experience of spinning up a CF env. I think it would benefit the Concourse deploy. Better would be to allow a CF to actually have two load balancers... or more!
